spec/databases/config.js

```diff
@@ -8,55 +8,73 @@ const dbs = {
 };
 
 if (process.env.POSTGRES_DSN) {
-  const postgres = new ConnectionString(process.env.POSTGRES_DSN);
+  const postgres = new ConnectionString(process.env.POSTGRES_DSN, {
+    user: 'postgres',
+    password: '',
+    database: 'sqlectron',
```

Comment:
shouldn't this (and other instances) be:
```suggestion
    path: ['sqlectron'],
```
?

Reply:
Oh my, my bad! :) Yes, the defaults for the database should be `path: ['sqlectron']`, and not `database: 'sqlectron'`. I guess it was just too late, and I started sleeping :)

Reply:
Corrected ;)